micro-benchmarks/DRB174-critical-orig-no.c

```diff
@@ -0,0 +1,41 @@
+/*
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+!!! Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+!!! and DataRaceBench project contributors. See the DataRaceBench/COPYRIGHT file for details.
+!!!
+!!! SPDX-License-Identifier: (BSD-3-Clause)
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+ */
+/**
+ * FT.C: This file is part of kernel of the NAS Parallel Benchmarks 3.0 FT suit.
+ * Intel Inspector can not correctly analysis the nowaite, and report a false postive.
+ * Tsan can not correctly analysis the critical, and report a false postive.
+*/
+
+#include <stdio.h>
+
+int main(int argh, char* argv[]){
+  int i;
+  double q[10], qq[10],x[20];
+
+  #pragma omp parallel for default(shared) private(i)
+  for (i = 0; i < 20; i++) x[i] = -1.0e99;
+  for (i = 0; i < 10; i++) qq[i] = 1.0;
```

Comment:
two loops (line 22 and 23) with same patterns: one is parallelized with "omp parallel for" and the other is not.   Why is this?